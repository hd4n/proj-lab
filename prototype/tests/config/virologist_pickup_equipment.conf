@items
ca c1

@fields
sh s1 c1

@virologists
v1 s1