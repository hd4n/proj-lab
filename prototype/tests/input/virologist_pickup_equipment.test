load virologist_pickup_equipment.conf
interact v1
getstats v1