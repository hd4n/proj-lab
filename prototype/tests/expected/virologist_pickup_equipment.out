Load: sikeres
interact: v1 s1

stats: v1
	fi_s1
	di_null
	st_false
	res_0.0
	re_false
	ma_10
	ref_0
	neededAmino_-1
	neededNucleo_-1
	co_null
	ma_null
	ag_null
	eq_c1
	eqa_null
	ef_null
